src/Uno.UI.Wasm/ts/WindowManager.ts

```diff
@@ -1340,6 +1340,14 @@
 			}
 			return rootElement === element || rootElement.contains(element);
 		}
+
+		public setCursor( cssCursor: string) {
+			const unoBody = document.getElementById("uno-body");
```

Comment:
`uno-body`: this element name is supplied in the constructor as `containerElementId`.

Reply:
i simply copied setupSplashScreen method, i will update it ! 

should i also update https://github.com/nventive/Uno/blob/0a54298294d493c1c822f790e54c847523fc04b0/src/Uno.UI.Wasm/ts/WindowManager.ts#L156 in the same commit ?

Reply:
Yes please ;-)

Reply:
well, turn out as it's a static function, it can't access it, but i guess when the WindowManager code will be refactored, it might move to a static const to solve that particular issue